lightbeam/send.py

```diff
@@ -26,6 +41,31 @@ def send(self):
             self.logger.info("finished processing endpoint {0}!".format(endpoint))
             self.logger.info("  (final status counts: {0}) ".format(self.lightbeam.status_counts))
             self.lightbeam.log_status_reasons()
+        
+        ### Create structured output results_file if necessary
+        if self.lightbeam.results_file:
+            self.end_timestamp = datetime.datetime.now()
+            self.metadata.update({"completed_at": self.end_timestamp.isoformat(timespec='microseconds')})
```

Comment:
This is a matter of personal preference, but why have all these update-statements as separate lines here?
```
self.metadata.update({
    "completed_at": self.end_timestamp.isoformat(timespec='microseconds'),
    "runtime_sec": (self.end_timestamp - self.start_timestamp).total_seconds(),
    "total_records_processed": sum(item['records_processed'] for item in self.metadata["resources"].values()),
    "total_records_skipped": sum(item['records_skipped'] for item in self.metadata["resources"].values()),
    "total_records_failed": sum(item['records_failed'] for item in self.metadata["resources"].values()),
})
```

Reply:
great suggestion, thanks!